go.mod

```diff
@@ -111,4 +111,12 @@ replace (
 	k8s.io/kubernetes => k8s.io/kubernetes v1.14.1
 )
 
-replace github.com/operator-framework/operator-lifecycle-manager => github.com/operator-framework/operator-lifecycle-manager v0.0.0-20190605231540-b8a4faf68e36
+replace (
+	// Indirect operator-sdk dependencies use git.apache.org, which is frequently
+	// down. The github mirror should be used instead.
+	// go.opencensus.io@v0.18.0
```

Comment:
What is this comment `// go.opencensus.io@v0.18.0` for? Same for Line 119.

Reply:
If you do `go mod graph | grep git.apache.org` you'll see both of these dependency versions using a different version of `git.apache.org/thrift` each.